catalog/spatial/geocoding/spatial-geocoding-feature/src/test/java/org/codice/ddf/spatial/geocoding/feature/TestCatalogFeatureQueryable.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.geocoding.feature;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.vividsolutions.jts.geom.Geometry;
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.federation.FederationException;
+import ddf.catalog.filter.FilterBuilder;
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.operation.impl.QueryImpl;
+import ddf.catalog.source.SourceUnavailableException;
+import ddf.catalog.source.UnsupportedQueryException;
+import java.util.Collections;
+import java.util.List;
+import org.codice.ddf.spatial.geocoding.FeatureQueryException;
+import org.codice.ddf.spatial.geocoding.GeoCodingConstants;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.opengis.feature.simple.SimpleFeature;
+
+public class TestCatalogFeatureQueryable {
+
+  private static final FilterBuilder FILTER_BUILDER = new GeotoolsFilterBuilder();
+
+  private static final String COUNTRY_CODE = "JAM";
+
+  private static final String WKT_STRING = "POLYGON((30 10, 10 20, 20 40, 40 40, 30 10))";
```

Comment:
Is that properly formatted? Should there be a space after "POLYGON"?

Reply:
I don't believe it matters since this is just being used in a mock response